apps/hobbies-helsinki/.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
.husky folder should also be merged to the root folder or just deleted

Reply:
fixed in #23